conda/meta.yaml

```diff
@@ -7,6 +7,7 @@ source:
   path: ../
 
 build:
+  noarch: python
   script: python -m pip install --no-deps --ignore-installed .
   script_env:
```

Comment:
I noticed that none of the scripts in the `bin` directory are used for unit testing. Still, are you sure you don't want to include any `entry points` for them here?